spec/martian_robot_spec.rb

```diff
@@ -1,82 +1,105 @@
 $LOAD_PATH << File.expand_path('. ./lib', __FILE__)
```

Comment:
Yes: when running the tests from inside the directory spec (now I'm running them from the main directory)